File: 42-alerts/python-sdk/alerts.py
import os, requests
API = "https://cost-katana-backend.store/api"
KEY = os.getenv("COST_KATANA_API_KEY")

def create_alert(name, threshold, alert_type):
    res = requests.post(f"{API}/alerts",
        json={"name": name, "threshold": threshold, "type": alert_type},
        headers={"Authorization": f"Bearer {KEY}"})
    res.raise_for_status()
    print(f"✅ Alert created: {res.json()['data']['id']}")
    return res.json()["data"]

File: 42-alerts/npm-package/alerts.ts
import axios from 'axios';
const API = 'https://cost-katana-backend.store/api'\;
const KEY = process.env.COST_KATANA_API_KEY;

export async function createAlert(name: string, threshold: number, type: string) {
  const res = await axios.post(`${API}/alerts`, {name, threshold, type},
    {headers: {'Authorization': `Bearer ${KEY}`}});
  console.log('✅ Alert created:', res.data.data.id);
  return res.data.data;
}
